tests/draft2020-12/unevaluatedItems.json

```diff
@@ -433,5 +433,134 @@
+                "data": [
+                    "test"
+                ],
+                "valid": true
+            },
+            {
+                "description": "uncle keyword evaluation is not significant",
+                "data": [
+                    "test",
+                    "test"
+                ],
+                "valid": false
+            }
+        ]
+    },
+    {
+        "description": "item is evaluated in an uncle schema to unevaluatedItems",
+        "schema": {
+            "type": "object",
+            "properties": {
+                "foo": {
+                    "type": "array",
+                    "prefixItems": [
+                        {
+                            "type": "string"
+                        }
+                    ],
+                    "unevaluatedItems": false
+                  }
+            },
+            "anyOf": [
+                {
+                    "properties": {
+                        "foo": {
+                            "prefixItems": [
+                                true,
+                                {
+                                    "type": "string"
+                                }
+                            ]
+                        }
+                    }
+                }
+            ]
+        },
+        "tests": [
+            {
+                "description": "no extra items",
+                "data": {
+                    "foo": [
+                        "test"
+                    ]
+                },
+                "valid": true
+            },
+            {
+                "description": "uncle keyword evaluation is not significant",
+                "data": {
+                    "foo": [
+                        "test",
+                        "test"
+                    ]
+                },
+                "valid": false
+            }
+        ]
+    },
+    {
+        "description": "item is evaluated in a nephew schema to unevaluatedItems",
+        "schema": {
+            "type": "array",
+            "prefixItems": [
+                {
+                    "type": "string"
+                }
+            ],
+            "unevaluatedItems": false,
+            "anyOf": [
+                {
+                    "prefixItems": [
+                        true,
+                        {
+                            "type": "string"
+                        }
+                    ]
+                }
+            ]
+        },
+        "tests": [
+            {
+                "description": "no extra items",
+                "data": [
+                    "test"
+                ],
+                "valid": true
+            },
+            {
+                "description": "all in-place applicators are evaluated before unevaluatedItems",
+                "data": [
+                    "test",
+                    "test"
+                ],
+                "valid": true
+            }
+        ]
```

Comment:
This also appears to be a duplicate of "unevaluatedItems with nested tuple" on line 120.

Reply:
agreed; removing.